.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+Thank you for contributing to Harbor!
+
+# Comprehensive Summary of your change
+
+# Issue being fixed
+Fixes #(issue)
+
+Please indicate you've done the following:
+- [ ] Well Written Title and Summary of the PR
+- [ ] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
```

Comment:
can an external person creating a PR even set label?

Reply:
the label can be added by the reviewers, and that's fair.